### PR TITLE
Update quiz9 to follow conventions

### PR DESCRIPTION
**Pull request by LeoCumpli21**
## Summary

This PR closes LeoCumpli21/PyZombis#23 by:
1. Adding subtitles to each exercise (*Ejercicio #*)
2. Changing each activecode snippet's id to be *q9_#*
2. Translating variable and function names to Spanish and writing them in underscore notation.
3. Checking python code is PEP8 compliant with `flake8`.
4. Adding a playwright test for ex 1 (watch the attached video).

## Checklist

- [x] Variables, functions and comments are translated to Spanish
- [x] Functions follow underscore notation
- [x] Spell check done & typos fixed
- [x] All python code is PEP8 compliant
- [x] Test coverage with Playwright implemented; locators are Pyhton code
- [x] Reviewers assigned (all peers & at least 1 mentor)

## Screenshots

(prefer Playwright recorded video or animated gif)

https://user-images.githubusercontent.com/80920858/121606853-87b6e880-ca14-11eb-8a91-643d24c6d09b.mp4

### CODE REVIEW

File: _sources/quiz/Quiz9.rst [hunk 1]
```diff
@@ -7,383 +7,524 @@ Quiz - 9
...
 
-   G. date_fashion |br|
-   Define una función ``date_fashion``. La función tomará dos dos argumentos ``mi_punt`` y ``pareja_punt``. |br|
-   Suponga que va un restaurante con su pareja. Los argumentos representan la puntuación de la ropa que llevan puesta del 0 al 10. |br|
-   Cuanto más alto es el puntaje, más elegante se han vestido. El puntaje de su ropa determinará si les dan una mesa en el restaurante o no:|br|
-   Si la puntuación de la ropa de uno de los dos es menor o igual a 2, no tendrán derecho a una mesa (0). |br|
-   Si las calificaciones son más altas, entonces si una de las dos es muy elegante (puntuación >= 8) Sí tendrán derecho a mesa (2). |br|
-   De lo contrario, la respuesta es tal vez (1). |br|
-   Entonces ``date_fashion`` regresa un número entre 0, 1 y 2: 0 = no, 1 = tal vez y 2 = sí |br| |br|
+   Define una función ``cita_de_lujo``. La función tomará dos dos parámetros
```
Comment (LeoCumpli21) [inline _sources/quiz/Quiz9.rst:330]:
```suggestion
   Define una función ``cita_de_lujo``. La función tomará dos parámetros 
```

File: _sources/quiz/Quiz9.rst [hunk 2]
```diff
@@ -7,383 +7,524 @@ Quiz - 9
...
-           self.assertEqual(alarm_clock(0, False), '10:00',"Esperado: 10:00")
-           self.assertEqual(alarm_clock(6, False), '10:00',"Esperado: 10:00")
-           self.assertEqual(alarm_clock(0, True), 'libre',"Esperado: libre")
-           self.assertEqual(alarm_clock(6, True), 'libre',"Esperado: libre")
-           self.assertEqual(alarm_clock(1, True), '10:00',"Esperado: 10:00")
-           self.assertEqual(alarm_clock(3, True), '10:00',"Esperado: 10:00")
-           self.assertEqual(alarm_clock(5, True), '10:00',"Esperado: 10:00")
-          
-         
+           self.assertEqual(poner_alarma(1, False), '7:00', "Esperado: 7:00")
```
Comment (y-martinez) [inline _sources/quiz/Quiz9.rst:519]: Trata de cambiar los ``' '`` por ``" "`` si quieres corre la herramienta ``black`` con las porciones de codigo de python para que arregle esas cositas.

File: _sources/quiz/Quiz9.rst [hunk 3]
```diff
@@ -7,383 +7,524 @@ Quiz - 9
...
-   Cuanto más alto es el puntaje, más elegante se han vestido. El puntaje de su ropa determinará si les dan una mesa en el restaurante o no:|br|
-   Si la puntuación de la ropa de uno de los dos es menor o igual a 2, no tendrán derecho a una mesa (0). |br|
-   Si las calificaciones son más altas, entonces si una de las dos es muy elegante (puntuación >= 8) Sí tendrán derecho a mesa (2). |br|
-   De lo contrario, la respuesta es tal vez (1). |br|
-   Entonces ``date_fashion`` regresa un número entre 0, 1 y 2: 0 = no, 1 = tal vez y 2 = sí |br| |br|
+   Define una función ``cita_de_lujo``. La función tomará dos dos parámetros 
+   ``mi_punt`` y ``pareja_punt``. Suponga que va un restaurante con su pareja. 
+   Los parámetros representan la puntuación de la ropa que llevan puesta del 0 al 10. 
+   Cuanto más alto es el puntaje, más elegante se han vestido. El puntaje de su ropa 
+   determinará si les dan una mesa en el restaurante o no:|br|
```
Comment (y-martinez) [inline _sources/quiz/Quiz9.rst:334]:
```suggestion
   determinará si les dan una mesa en el restaurante o no, de acuerdo a las siguientes reglas:|br|
```

File: _sources/quiz/Quiz9.rst [hunk 4]
```diff
@@ -7,383 +7,524 @@ Quiz - 9
...
+
+   - Si la puntuación de la ropa de uno de los dos es menor o igual a 2, 
+     no tendrán derecho a una mesa (``0``). |br|
+   - Si las calificaciones son más altas, entonces si una de las dos es muy 
+     elegante (puntuación >= 8) sí tendrán derecho a mesa (``2``). |br|
+   - De lo contrario, la respuesta es tal vez (``1``). |br|
+  
+   Entonces ``cita_de_lujo`` devuelve un número entre 0, 1 y 2: 0 = no, 1 = tal vez y 2 = sí |br| |br|
```
Comment (y-martinez) [inline _sources/quiz/Quiz9.rst:342]:
```suggestion
   Entonces ``cita_de_lujo`` devuelve un número entre 0, 1 y 2 que significan No, Tal vez y Si respectivamente |br| |br|
```

File: _sources/quiz/Quiz9.rst [hunk 5]
```diff
@@ -7,383 +7,524 @@ Quiz - 9
...
    :nocodelens:
 
-   H. squirrel_play |br|
-   El equipo de fútbol *ardillas* normalmente juega cuando la temperatura está entre 60 y 90 grados Fahrenheit, pero cuando es verano la temperatura superior a la que juegan las *ardillas* es de 100 grados Fahrenheit en lugar de 90. |br|
-   Defina una función ``squirrel_play`` que tome como argumentos la temperatura y ``True`` si es verano o ``False`` si no lo es. La función debe regresar ``True`` si las *ardillas* juegan o ``False`` si no. |br| |br|
+   El equipo de fútbol *ardillas* normalmente juega cuando la temperatura 
+   está entre 60 y 90 grados Fahrenheit, pero cuando es verano la temperatura 
+   superior a la que juegan las *ardillas* es de 100 grados Fahrenheit en lugar de 90.
+   Defina una función ``juegan_ardillas`` que tome 2 parámetros, ``temp`` representando 
+   la temperatura, y ``es_verano``, que puede ser ``True`` si es verano o ``False``.
```
Comment (y-martinez) [inline _sources/quiz/Quiz9.rst:386]:
```suggestion
   la temperatura, y ``es_verano``, que puede ser ``True`` si es verano o ``False`` si no lo es. 
```

File: _sources/quiz/Quiz9.rst [hunk 6]
```diff
@@ -7,383 +7,524 @@ Quiz - 9
...
-   ``alarm_clock(1, False)`` -> '7:00' |br|
-   ``alarm_clock(1, True)`` -> '10:00' |br|
-   ``alarm_clock(6, True)`` -> 'libre' |br|
-   ``alarm_clock(0, False)`` -> '10:00' |br|
 
+   - Días de la semana: '10:00' |br|
+   - Fines de semana: 'libre' |br| |br|
+ 
+   Ejemplos: |br|
+   ``poner_alarma(1, False)`` -> ``'7:00'`` |br|
```
Comment (y-martinez) [inline _sources/quiz/Quiz9.rst:503]: Igual aqui las dobles comillas

**Review by y-martinez — CHANGES_REQUESTED** [review]
A few comments but in general it is good 👍🏼

File: _sources/quiz/Quiz9.rst [hunk 7]
```diff
@@ -7,383 +7,524 @@ Quiz - 9
...
+   ``poner_alarma(1, True)`` -> ``'10:00'`` |br|
+   ``poner_alarma(6, True)`` -> ``'libre'`` |br|
+   ``poner_alarma(0, False)`` -> ``'10:00'`` |br|
```
Comment (LeoCumpli21) [inline _sources/quiz/Quiz9.rst:506]:
```suggestion
   ``poner_alarma(1, False)`` -> ``"7:00"`` |br|
   ``poner_alarma(1, True)`` -> ``"10:00"`` |br|
   ``poner_alarma(6, True)`` -> ``"libre"`` |br|
   ``poner_alarma(0, False)`` -> ``"10:00"`` |br|
```

File: _sources/quiz/Quiz9.rst (hunk 2 above)
Comment (LeoCumpli21) [reply _sources/quiz/Quiz9.rst]: Listo! muy buena herramienta btw 👍🏽

**Review by y-martinez — APPROVED** [review]
LGTM 🚀

**Review by reingart — APPROVED** [review]
LGTM

**Pull request merged**